pkg/controller/vsphere/reconciler_test.go

```diff
@@ -992,33 +992,15 @@ func TestDelete(t *testing.T) {
 }
 
 func TestCreate(t *testing.T) {
-	model, session, server := initSimulator(t)
-	defer model.Remove()
```

Comment:
Is there any reason why this change is in this commit?

Reply:
Thanks for finding it. I don't remember why it's like this :D